<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPassword</class>
 <widget class="QWidget" name="ForgetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: #1A1A2A*/
/*background-color: rgb(138, 138, 255);*/
background-color: rgb(114, 159, 207);</string>
  </property>
  <widget class="QPushButton" name="savePasswordButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>101</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(255, 132, 77);*/
QPushButton {
    background-color: rgb(255, 132, 77); /* Orange background */
    color: rgb(0, 0, 0); /* Black text */
    border: 2px solid rgb(205, 92, 92); /* Indian Red border */
    padding: 10px 20px; /* Padding for better appearance */
    border-radius: 5px; /* Rounded corners for a modern look */
    font-size: 16px; /* Adjust font size */
    font-weight: bold; /* Bold text */
    transition: background-color 0.3s ease, border-color 0.3s ease; /* Smooth transition */
}

QPushButton:hover {
    background-color: rgb(255, 69, 0); /* Red-Orange on hover */
}

QPushButton:focus {
    border-color: rgb(255, 255, 255); /* White border on focus */
    box-shadow: 0 0 5px rgba(255, 255, 255, 0.5); /* Optional shadow on focus */
}
</string>
   </property>
   <property name="text">
    <string>Save </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>421</width>
     <height>202</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>   Security Question</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>48</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="forgotPasswordSecurityQuestioncomboBox">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Please select Security Question</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #5A5A5A;  /* Border color */
    border-radius: 5px;         /* Rounded corners */
    padding: 5px;               /* Padding inside the combobox */
    background-color: #F0F0F0;  /* Background color */
    color: #333333;             /* Text color */
    font-size: 14px;            /* Font size */
    min-width: 150px;           /* Minimum width */
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;                /* Width of the drop-down button */
    border-left: 1px solid #5A5A5A; /* Border between combobox and drop-down button */
}

QComboBox::down-arrow {
    image: url(:/images/down_arrow.png); /* Path to custom arrow image */
    width: 10px;               /* Width of the arrow */
    height: 10px;              /* Height of the arrow */
}

QComboBox QAbstractItemView {
    border: 1px solid #5A5A5A; /* Border for the drop-down list */
    selection-background-color: #3399FF; /* Background color for selected item */
    selection-color: #FFFFFF;   /* Text color for selected item */
    background-color: #FFFFFF;  /* Background color for the list */
    color: #333333;             /* Text color for the list */
}

QComboBox::item {
    background-color: #FFFFFF;  /* Background color for items */
    color: #333333;             /* Text color for items */
    padding: 5px;               /* Padding for items */
}

QComboBox::item:selected {
    background-color: #3399FF;  /* Background color for selected item */
    color: #FFFFFF;             /* Text color for selected item */
}
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="forgotPasswordSecurityQuestionlineEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Enter answer of Security Question</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/*background-color: rgb(158, 109, 97,0%);*/
 background-color: rgba(255, 255, 255, 0.8);
color: rgb(0, 0, 0);
border: 1px solid rgb(100, 100, 100);
 padding: 5px;
 border-radius: 5px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter your answer here....</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>New Password</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="newPasswordLineEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Please enter User Id</string>
          </property>
          <property name="toolTipDuration">
           <number>10</number>
          </property>
          <property name="styleSheet">
           <string notr="true">/*background-color: rgb(158, 109, 97,0%);*/
 background-color: rgba(255, 255, 255, 0.8);
color: rgb(0, 0, 0);
border: 1px solid rgb(100, 100, 100);
 padding: 5px;
 border-radius: 5px;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>New Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Confirm Password</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="confirmPasswordlineEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Please enter First Name</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/*background-color: rgb(158, 109, 97,0%);*/
 background-color: rgba(255, 255, 255, 0.8);
color: rgb(0, 0, 0);
border: 1px solid rgb(100, 100, 100);
 padding: 5px;
 border-radius: 5px;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Confirm Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
